content/articles/2019-08-gargle-googledrive-1-0-0.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: gargle and googledrive v1.0.0
+slug: gargle-googledrive-1-0-0
+author: Jenny Bryan
+description: >
+  gargle and googledrive v1.0.0 are now on CRAN.
+date: '2019-08-21'
+categories: [package]
+tags:
+  - gargle
+  - googledrive
+photo:
+  url: https://flic.kr/p/oe2LJ1
```

Comment:
sheer curiosity: is there some reason for this particular picture?

Reply:
It was impossible to find a good-looking, un-creepy photo relevant to gargling with an acceptable license.

So I resorted to line art from an old medical text 😃

Reply:
I'm open to other ideas, btw!

Reply:
a few minutes of searching makes me strongly believe your analysis. 😄